.travis.yml

```diff
@@ -5,97 +5,93 @@ cache: ccache
 sudo: required
 
 addons:
-    apt:
-        packages:
-            - asciidoc
-            - libxslt1-dev
-            - xsltproc
-            - docbook-xml
-            - docbook-xsl
-            - gperf
-            - elfutils
-            - zlib1g-dev
-            - libmemcached-dev
-            - libmemcached-tools
-            - memcached
-            - mingw32
-            - mingw32-binutils
-            - clang    # scan-build
+  apt:
+    packages:
+      - asciidoc
+      - libxslt1-dev
+      - xsltproc
+      - docbook-xml
+      - docbook-xsl
+      - gperf
+      - elfutils
+      - zlib1g-dev
+      - libmemcached-dev
+      - libmemcached-tools
+      - memcached
+      - mingw32
+      - mingw32-binutils
+      - clang
 
 os:
-    - linux
-    - osx
+  - linux
+  - osx
 
 compiler:
-    - clang
-    - gcc
+  - clang
+  - gcc
 
 env:
   global:
     - XML_CATALOG_FILES=/usr/local/etc/xml/catalog
 
 matrix:
-    include:
-        - os: linux
-          compiler: gcc
-          sudo: required
-          dist: trusty
-          env: FEATURES="--enable-memcached"
-        - os: linux
-          compiler: clang
-          env: FEATURES="--enable-memcached"
-#        - os: linux
-#          compiler: i586-mingw32msvc-gcc
-#          env: HOST="--host=i586-mingw32msvc" TEST="test/main.exe"
-        - os: linux
-          compiler: clang
-          env: CFLAGS="-fsanitize=undefined" ASAN_OPTIONS="detect_leaks=0"
-        - os: linux
-          compiler: clang
-          env: CFLAGS="-fsanitize=address -g" ASAN_OPTIONS="detect_leaks=0"
-        - os: linux
-          compiler: clang
-          env: PATH="/usr/bin:$PATH" TEST=analyze
-        - os: linux
-          compiler: gcc
-          env: CUDA=8.0.61-1
-        - os: osx
-          osx_image: xcode8.3
-          compiler: clang
-          env: FEATURES=" --enable-memcached --with-bundled-zlib"
-        - os: osx
-          osx_image: xcode9
-          compiler: clang
-          env: FEATURES="--enable-memcached --with-bundled-zlib"
-
-    exclude:
-        - os: osx
-          compiler: gcc
+  include:
+    - os: linux
+      compiler: gcc
+      sudo: required
+      dist: trusty
+      env: FEATURES="--enable-memcached"
+    - os: linux
+      compiler: clang
```

Comment:
what does that mean? Does it compile with clang, or for clang? Does ccache have to be compiled _for_ certain compiler?

Reply:
Documentation of travis says that are tests suits agains clang/gcc: https://docs.travis-ci.com/user/languages/c/

I removed other linux entries - Left only those added by Martin.

Reply:
It's still not entirely clear to me, but I can figure it out later not to block this PR.